target/linux/realtek/dts-5.10/rtl9302_zyxel_xgs1010-12.dts

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: (GPL-2.0-or-later or MIT)
+/dts-v1/;
+
+#include "rtl9302_zyxel_xgs1210-12.dts"
+
+/ {
+	compatible = "zyxel,xgs1010-12", "realtek,rtl838x-soc";
+	model = "Zyxel XGS1010-12 Switch";
+};
+
+&partitions {
+	compatible = "fixed-partitions";
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	partition@300000 {
```

Comment:
made a little booboo here, this all needs to go:
```suggestion
	partition@300000 {
```
while trying to boot the 1210 firmware on my 1010, I noticed this sloppyness (my development leftover that I hadn't cleaned yet). This led me to find a typo bug in the 12xx dtsi; which this code here actually 'fixes' (for the wrong reasons).

Sorry for not testing all 3 variants on my device!